rubies/ruby/CVE-2015-1855.yml

```diff
@@ -15,3 +15,4 @@ patched_versions:
   - ~> 2.0.0.645
   - ~> 2.1.6
   - ~> 2.2.2
+  - ">= 2.3.0"
```

Comment:
I was just following the convention of this commit: 51c76a755d2955ca1e621dd0c4b9a2de3134d9c5 . But yes, I like `">= 2.2.2"` better. I'll update the PR.